Cartfile

```diff
@@ -1,2 +1,2 @@
-github "raphaelmor/Polyline" ~> 5.0
+github "MaximAlien/Polyline" "master"
```

Comment:
New release of Polyline with updated deployment targets is needed. PR can be found here: https://github.com/raphaelmor/Polyline/pull/78.